projects/noloan/firebase/src/main/java/hasadna/noloan/firebase/DbMessages.java

```diff
@@ -32,7 +32,7 @@ public static DbMessages getInstance() {
 
   // Search messages by sender & body. Return -1 if no message found
   public int findBySms(SmsMessage smsMessage) {
-    return findSmsByBody(smsMessage, messages);
+    return findSms(smsMessage, messages);
```

Comment:
https://github.com/hasadna/monorepo/pull/246#discussion_r344371555

`s/findBySms/findSms` ?
For consistency, this is confusing IMO...

Reply:
We have `findSmsById` So I think for consistency this suppose to be `findSmsBy___` . the question is what is this ___..

Reply:
https://github.com/hasadna/monorepo/pull/246#discussion_r343853004
That's why I said we can agree on `findSms()` as a function that finds by body & sender.
Aside to that for the need of finding a message by its id, have another function: `findSmsById()`

Having all:
`public int findBySms()`
`public int findById()`
`public static int findSms`
`public static int findSmsById()`

For me it sounds confusing and not necessary.
So please just write some "brief" summary in the channel at the end, about the new usage of the functions you changed ok? It'll be easier to follow...

Reply:
The statics functions are for outside use. 
the non-statics function are using the static function on the DB list

and you can search by id or by body and sender

Reply:
Ok, now it's becoming more clear... :+1: